R/accounts.R

```diff
@@ -235,7 +235,11 @@ setAccountInfo <- function(name, token, secret) {
     stop(stringParamErrorMessage("secret"))
 
   # create connect client
-  serverInfo <- shinyappsServerInfo()
+  if (cloud) {
+    serverInfo <- cloudServerInfo()
+  } else {
+    serverInfo <- shinyappsServerInfo()
```

Comment:
Should we create a stricter separation between shinyapps.io and cloud? Just below this change, for example, is still attempting to use `lucidClient` even when `cloud = TRUE`.

Reply:
I took another pass at this and turned `lucidClient` into `lucidClientForAccount`. This ends up returning a client that is either configured to talk to shinyapps.io or one that is targeting rstudio.cloud depending on the account passed in which I think ends up cleaning up the references in other areas of the code base.